docs/release-notes/Percona-Everest-1.5.0-(2025-02-07).md

```diff
@@ -0,0 +1,160 @@
+
+- [EVEREST-1674](https://perconadev.atlassian.net/browse/EVEREST-1674): The message **Enforce did not pass** appeared randomly in the UI. Additionally, databases in the UI sometimes disappeared and then reappeared after a few seconds. We have now resolved the issue.
+
+
+- [EVEREST-1724](https://perconadev.atlassian.net/browse/EVEREST-1724): Sharding no longer resets to its default setting (disabled) when navigating back in the database creation wizard.
+
+
+- [EVEREST-1728](https://perconadev.atlassian.net/browse/EVEREST-1728): The database list now refreshes automatically whenever new databases are deployed in recently created namespaces, and this update is reflected across various browser sessions.
+
+- [EVEREST-1758](https://perconadev.atlassian.net/browse/EVEREST-1758): The **Create Database** button on the UI was inaccessible until the page was manually refreshed after adding a namespace using the `everestctl namespaces add` command. The issue has been resolved now.
+
+- [EVEREST-1729](https://perconadev.atlassian.net/browse/EVEREST-1729): Installation fails if Percona Everest 1.3.0 release CLI is used to install 1.4.0.
+
+- [EVEREST-1735](https://perconadev.atlassian.net/browse/EVEREST-1735): We have fixed an issue where incorrect information appeared on the **PITR Edit** button when PITR was enabled.
+
+
+- [EVEREST-1800](https://perconadev.atlassian.net/browse/EVEREST-1800): We fixed an issue where users could not create a DB cluster because the DB version was not displayed with a specific RBAC policy.
+
+- [EVEREST-1801](https://perconadev.atlassian.net/browse/EVEREST-1801): The **Create DB cluster** option was not visible on the UI if the user did not have the permissions for all the database engines. The issue has been resolved now.
+
+- [EVEREST-1802](https://perconadev.atlassian.net/browse/EVEREST-1802): A given namespace was not visible on the UI if the user lacked permissions for all database engines in that namespace. This issue has now been resolved.
+
+
+- [EVEREST-1803](https://perconadev.atlassian.net/browse/EVEREST-1803): 
+We have fixed an issue that prevented users from editing or adding monitoring to an existing DB cluster with a specific RBAC policy.
+
+- [EVEREST-1804](https://perconadev.atlassian.net/browse/EVEREST-1804): The Operator Upgrade option was previously not visible on the UI when users didn't have access to all DB clusters in a namespace. This issue has now been resolved.
+
+- [EVEREST-1811](https://perconadev.atlassian.net/browse/EVEREST-1811) The everest-operator now restarts seamlessly when a DB operator is installed for the first time.
+
+
+## Known limitations
+
+Operator versions 1.19.0 and 1.19.1 are set with a recommended MongoDB version of 7.0 due to potential issues with point-in-time recovery on MongoDB 8.0 when sharding is enabled, and the Operator version is 1.19.x. Thus, upgrading to Operator 1.19.0 or 1.19.1 is not advisable if you run sharded MongoDB 8.0 clusters.
```

Comment:
Since we only support 1.19.1, I suggest to only mention that version. Also, MongoDB version 8 has only been added in this version so the last sentence doesn't make sense.
```suggestion
PSMDB Operator version 1.19.1 added support for MongoDB version 8.0. However, due to potential issues with point-in-time recovery on MongoDB 8.0 when sharding is enabled, the recommended MongoDB version is still 7.0.
```